### ISSUE TITLE
☁️FrontDoor to Cloudflare Part 2/2 - sswdory.com & ssw.com

### ISSUE THREAD

**Issue by amankumarrr** [SSWConsulting/SSW.Website#2781]
@chrisschultzssw 

As per my conversation with @wicksipedia @jeoffreyfischer, we have successfully migrated tfs365.com and ssw.com.au to Cloudflare, including all necessary redirects. With this milestone achieved, our next steps are to move the remaining domains to Cloudflare to enable the complete removal of the Front Door.

```[tasklist]
### Tasks
- [x] CDN - Move ssw.com and its aossicated redirects
- [x] CDN - Move sswdory.com. and its associated redirects
- [x] Decommission FrontDoor
```

**Comment by Calinator444** [comment]
@wicksipedia 

Is this a blocker for switching off the VM? The redirects for ssw.com.au are moved. However as far as I'm aware we haven't set a preferred URL on for sswdory.com

**Comment by wicksipedia** [comment]
> [<img alt="" width="16" height="16" src="https://avatars.githubusercontent.com/u/600044?s=64&amp;u=94fb0c525fe0e8c3dbf92d9146609c34357ec737&amp;v=4">@wicksipedia](https://github.com/wicksipedia)
> 
> Is this a blocker for switching off the VM? The redirects for ssw.com.au are moved. However as far as I'm aware we haven't set a preferred URL on for sswdory.com

no, it'll be a blocker for switching off frontdoor

**Comment by amankumarrr** [comment]
We've found some logs showing traffic still hitting our website through Front Door, despite having moved our server names to Cloudflare. We're unsure why this is happening and need to investigate further.

**Comment by amankumarrr** [comment]
All redirects are tested and seems to be working fine for both domains.

**Comment by amankumarrr** [comment]
We’ve noticed some traffic on FrontDoor that ideally shouldn’t be there. After investigating the logs, we plan to decommission it.

**Comment by amankumarrr** [comment]
I’ve developed a theory regarding the traffic showing up in the FrontDoor logs. It appears that most of the links are coming from China, and since Cloudflare is blocked in China, it seems the DNS requests are being routed to our FrontDoor, possibly via internal DNS.
 
I initially thought the traffic might be related to our CodeAuditor, but after investigating further, I found that the source country was China, while we run CodeAuditor from our pipeline, which shouldn’t originate from there. Additionally, I noticed a user agent that was already linked to a known user, confirming it wasn’t CodeAuditor.



![Image](https://github.com/user-attachments/assets/b419070d-88f8-456a-a074-b3230fa88c0e)

**Figure: Logs from FrontDoor for the past 3 days**
